volmdlr/edges.py

```diff
@@ -2260,8 +2260,25 @@ def get_geo_lines(self, tag: int, start_point_tag: int, center_point_tag: int, e
             str(center_point_tag) + ', ' + str(end_point_tag) + '};'
 
     def get_geo_points(self):
+        """
+        Gets the points that define an Arc to use them in a .geo file.
+
+        :return: A list of caracteristic arc points
+        :rtype: List
+
+        """
         return [self.start, self.center, self.end]
 
+    def reverse(self):
+        """
+        Gets the reverse version of an arc.
+
+        :return: An arc
+        :rtype: Arc
+        """
+
+        return self.__class__(start=self.end, interior=self.interior, end=self.start)
```

Comment:
There exists a method like this inside Arc2D. Could you delete it then, please? So this one replaces it.